sdk/python/flet/page.py

```diff
@@ -102,6 +103,9 @@ def convert_keyboard_event(e):
         self.__on_disconnect = EventHandler()
         self._add_event_handler("disconnect", self.__on_disconnect.handler)
 
+        # Querystring
+        self.query = QueryString(page=self)
```

Comment:
1) looks like route is getting parsed on initial page load only? It needs to be parsed on `route_change` event too, right?
2) Let's use a private `page.__query` to store parsed query and then expose `page.query` as a property, like anything else.

Reply:
Hello, been busy recently due to university, so I was not able to refactor the code.

As for code, what I did is the following:
 I initialize `QueryString` object as the page is initialized. This case, every time route is changed, `QueryString` object must be updated to contain always up-to-date route. To contain current URL, I implemented __call__ method which is responsible for updating current URL inside of `QueryString` object every time `go` method is used.

As for `route_change` event, I consider it to be good use for updating current URL which I'll try soon.

As for page.query as property, I think that's a really good idea. I'll work on that soon.

Reply:
yep. `route_change` is called when you change URL manually in the browser address bar.